documentation/asciidoc/computers/raspberry-pi/display-parallel-interface.adoc

```diff
@@ -8,11 +8,323 @@ This interface is controlled by the GPU firmware and can be programmed by a user
 
 One of the alternate functions selectable on bank 0 of the Raspberry Pi GPIO is DPI (Display Parallel Interface) which is a simple clocked parallel interface (up to 8 bits of R, G and B; clock, enable, hsync, and vsync). This interface is available as alternate function 2 (ALT2) on GPIO bank 0:
 
-image::images/dpi-altfn2.png[DPI Alternate GPIO function]
+[cols=2]
+|===
+|GPIO |ALT2
+
+|GPIO0
+|PCLK
+
+|GPIO1
+|DE
+
+|GPIO2
+|LCD_VSYNC
+
+|GPIO3
+|LCD_HSYNC
+
+|GPIO4
+|DPI_D0
+
+|GPIO5
+|DPI_D1
+
+|GPIO6
+|DPI_D2
+
+|GPIO7
+|DPI_D3
+
+|GPIO8
+|DPI_D4
+
+|GPIO9
+|DPI_D5
+
+|GPIO10
+|DPI_D6
+
+|GPIO11
+|DPI_D7
+
+|GPIO12
+|DPI_D8
+
+|GPIO13
+|DPI_D9
+
+|GPIO14
+|DPI_D10
+
+|GPIO15
+|DPI_D11
+
+|GPIO16
+|DPI_D12
+
+|GPIO17
+|DPI_D13
+
+|GPIO18
+|DPI_D14
+
+|GPIO19
+|DPI_D15
+
+|GPIO20
+|DPI_D16
+
+|GPIO21
+|DPI_D17
+
+|GPIO22
+|DPI_D18
+
+|GPIO23
+|DPI_D19
+
+|GPIO24
+|DPI_D20
+
+|GPIO25
+|DPI_D21
+
+|GPIO26
+|DPI_D22
+
+|GPIO27
+|DPI_D23
+|===
 
 NOTE: There are various ways that the colour values can be presented on the DPI output pins in either 565, 666, or 24-bit modes (see the following table and the `output_format` part of the `dpi_output_format` parameter below):
 
-image::images/dpi-packing.png[DPI Colour Output]
+[cols=26]
+|===
+.2+h|*Mode* .2+h|*RGB bits* 24+h|*GPIO*
+h|*27* h|*26* h|*25* h|*24* h|*23* h|*22* h|*21* h|*20* h|*19* h|*18* h|*17* h|*16* h|*15* h|*14* h|*13* h|*12* h|*11* h|*10* h|*9* h|*8* h|*7* h|*6* h|*5* h|*4*
```

Comment:
@andrum99 May be erase `h` and follow the rules `e.g ) 24+^|GPIO` in front of the table you want to sort and it works!
@mudge Thank you for your contribution !

Reply:
> @andrum99 May be erase `h` and follow the rules `e.g ) 24+^|GPIO` in front of the table you want to sort and it works!

I don't know what you mean - if you have fixed it, can you provide a PR?